config.js

```diff
@@ -1,99 +1,93 @@
-var testRpUrl = process.env.VERIFY_TEST_RP_URL || 'http://localhost:50130/test-rp'
-var frontendDomain = process.env.VERIFY_FRONTEND_DOMAIN || 'http://localhost:50300'
+let testRpUrl = process.env.VERIFY_TEST_RP_URL || 'http://localhost:50130/test-rp'
+let frontendDomain = process.env.VERIFY_FRONTEND_DOMAIN || 'http://localhost:50300'
 
 module.exports = {
-  "id": "backstop_verify",
-  "viewports": [
-    {
-      "label": "phone",
-      "width": 320,
-      "height": 480
+    "id": "backstop_verify",
+    "viewports": [
+        {
+            "label": "phone",
+            "width": 320,
+            "height": 480
+        },
+        {
+            "label": "tablet",
+            "height": 768,
+            "width": 1024
+        },
+        {
+            "label": "desktop",
+            "width": 1366,
+            "height": 768
+        }
+    ],
+    "misMatchThreshold": 0,
+    "requireSameDimensions": false,
+    "onBeforeScript": "onBefore.js",
+    "onReadyScript": "onReady.js",
+    "scenarios": [
+        {
+            "label": "Start",
+            "url": frontendDomain + "/start",
+            "getSession": testRpUrl,
+        },
+        {
+            "label": "Start with previous IDP",
+            "url": frontendDomain + "/start",
+            "getSession": testRpUrl,
+            "previouslySignedInWithIDP": frontendDomain + "/sign-in",
+        },
+        {
+            "label": "About",
+            "url": frontendDomain + "/about",
+            "getSession": testRpUrl,
+            "sortAboutCompanies": true
+        },
+        {
+            "label": "Will it work for me",
+            "url": frontendDomain + "/will-it-work-for-me",
+            "getSession": testRpUrl,
+        },
+        {
+            "label": "Will it work for me - validation",
+            "url": frontendDomain + "/will-it-work-for-me",
+            "getSession": testRpUrl,
+            "submitForm": true
+        },
+        {
+            "label": "About documents",
+            "url": frontendDomain + "/about-documents",
+            "getSession": testRpUrl,
+        },
+        {
+            "label": "Choose a certified company",
+            "url": frontendDomain + "/choose-a-certified-company",
+            "getSession": testRpUrl,
+            "selectDocuments": true,
```

Comment:
as above, maybe change this key to `aboutDocuments` or `confirmAboutDocuments` maybe.

Reply:
Good point but as above I'll just remove this completely in the next PR